djangoapp/app/views.py

```diff
@@ -1,6 +1,5 @@
 import os, platform, configparser
 from django.shortcuts import render
-from .forms import ProfileForm
 from .models import ProfilesTable
 from django_tables2 import RequestConfig
```

Comment:
I'm still not convinced using `django_tables2` is a good idea. We should keep this app as simple as possible with minimal third party dependencies, and I don't think responsive HTML tables is a big enough deal to warrant adding an extra package. Also, I'm pretty sure the gov.uk frontend toolkit handles responsive tables pretty well already.